packages/application/src/plugins/rise.ts

```diff
@@ -273,37 +278,21 @@ namespace Rise {
   }
 
   /* Register commands */
-  function smartExec() {
-    // TODO
-    // is it really the selected cell that matters ?
-    // let smart_exec = Jupyter.notebook.get_selected_cell().smart_exec;
-    // if (smart_exec == 'smart_exec_slide') {
-    //   Jupyter.notebook.execute_selected_cells();
-    // } else if (smart_exec == "smart_exec_fragment") {
-    //   // let's see if the next fragment is visible or not
-    //   let cell = Jupyter.notebook.get_selected_cell();
-    //   let fragment_div = cell.smart_exec_next_fragment;
-    //   let visible = $(fragment_div).hasClass('visible');
-    //   if (visible) {
-    //     Jupyter.notebook.execute_cell_and_select_below();
-    //   } else {
-    //     Jupyter.notebook.execute_selected_cells();
-    //   }
-    // } else {
-    //   Jupyter.notebook.execute_cell_and_select_below();
-    // }
```

Comment:
I know it is not great but could you keep that old piece of code as the smart execution feat' is missing and that snippet will likely be useful when restoring it.